src/libNode/Node.cpp

```diff
@@ -155,7 +155,8 @@ bool Node::DownloadPersistenceFromS3() {
   // TBD - find better way to capture the exit status of command
   string excludembtxns = LOOKUP_NODE_MODE ? "false" : "true";
   SysCommand::ExecuteCmdWithOutput(
-      "./downloadIncrDB.py " + STORAGE_PATH + "/ " + excludembtxns, output);
+      "/run/zilliqa/download_incr_DB.py " + STORAGE_PATH + "/ " + excludembtxns,
```

Comment:
The binary path should not be hard-coded like this. Assuming the user will put the binary in the same location as we do is not ideal unless you have told users to do so.

Reply:
ok, I'll put the binary path to `constants.xml`